go.mod

```diff
@@ -3,16 +3,28 @@ module github.com/ipfs/ipfs-ds-convert
 go 1.12
 
 require (
-	github.com/ipfs/go-datastore v0.4.4
+	github.com/DataDog/zstd v1.4.5 // indirect
+	github.com/dgraph-io/badger/v2 v2.2007.2 // indirect
+	github.com/dgraph-io/ristretto v0.0.3 // indirect
+	github.com/dgryski/go-farm v0.0.0-20200201041132-a6ae2369ad13 // indirect
+	github.com/golang/snappy v0.0.2 // indirect
+	github.com/google/uuid v1.1.2 // indirect
+	github.com/ipfs/go-datastore v0.4.5
 	github.com/ipfs/go-ds-badger v0.2.4
-	github.com/ipfs/go-ds-flatfs v0.4.4
+	github.com/ipfs/go-ds-badger2 v0.1.0
+	github.com/ipfs/go-ds-flatfs v0.4.5
 	github.com/ipfs/go-ds-leveldb v0.4.2
 	github.com/ipfs/go-ds-measure v0.1.0
-	github.com/ipfs/go-fs-lock v0.0.4
-	github.com/ipfs/go-ipfs v0.5.0-rc2
-	github.com/ipfs/go-ipfs-config v0.5.2
+	github.com/ipfs/go-fs-lock v0.0.6
+	github.com/ipfs/go-ipfs v0.7.0
+	github.com/ipfs/go-ipfs-config v0.9.0
```

Comment:
Let's have a commit that updates the go mod to the latest version of go-ipfs before we do the badger update